### PR TITLE
#494 atomsphere conversion

### LINKED ISSUE #494: Add inverse conversion of the atomospheric  refraction
**Motivation**

- ...

**What should be realised**

```python
...
```

### PR DESCRIPTION
**Pull request by r-yamada1998**
**Checks**

- [x] Debugging completed
- [ ] Please debug me

**Related Issue**

This PR closes #494

### CODE REVIEW

File: neclib/coordinates/pointing_error/omu1p85m.py [hunk 1]
```diff
@@ -88,5 +89,146 @@ def offset(self, az: u.Quantity, el: u.Quantity) -> Tuple[u.Quantity, u.Quantity
         # The above is defined as (refracted + offset = apparent), so reverse the sign
         return -1 * dAz, -1 * dEl
```
Comment (TakutoIto) [inline neclib/coordinates/pointing_error/omu1p85m.py:90]: No need to revers the sign of dAz,dEl

File: neclib/coordinates/pointing_error/pointing_error.py [hunk 2]
```diff
@@ -215,7 +279,9 @@ def refracted_to_apparent(
 
         """
         _az, _el = get_quantity(az, el, unit=unit)
+        print(type(_az), type(_el))
         dAz, dEl = self.offset(_az, _el)
+        print(type(dAz), type(dEl))
         return _az - dAz, _el - dEl
```
Comment (TakutoIto) [inline neclib/coordinates/pointing_error/pointing_error.py:285]: reverse the sign of dAz,dEl

File: neclib/coordinates/pointing_error/pointing_error.py [hunk 3]
```diff
@@ -117,6 +125,61 @@ def offset(self, az: u.Quantity, el: u.Quantity) -> Tuple[u.Quantity, u.Quantity
...
+        ...
+
+    def inverse_atmospheric_refraction(
+        self,
+        az: Union[u.Quantity, DimensionLess],
+        el: Union[u.Quantity, DimensionLess],
+        phpa: u.hPa,
+        tc: u.deg_C,
+        rh: float,
+        wl: u.micron,
```
Comment (TakutoIto) [inline neclib/coordinates/pointing_error/pointing_error.py:163]: phpa -> pressure 
tc -> temperature
rh -> relative_humidity
wl -> obs_wl

**Review by TakutoIto — CHANGES_REQUESTED** [review]
Reverse sign of dAz, dEl.
Change name of parameter about weather.
There are few test patterns.

File: neclib/coordinates/pointing_error/omu1p85m.py [hunk 4]
```diff
@@ -86,7 +38,110 @@ def offset(self, az: u.Quantity, el: u.Quantity) -> Tuple[u.Quantity, u.Quantity
...
+                + g1
+                + 1
+            )
+            return f
+
+        dAz, dEl = self.apply_offset(az, el)
+        az0 = az - dAz
+        el0 = el - dEl
+        x0 = np.array([az0.deg, el0.deg])
+
```
Comment (TakutoIto) [inline neclib/coordinates/pointing_error/omu1p85m.py:140]: apply offset return az+dAz, el+dEl.
So, az0, el0 = self.apply_offset(az,el)

**Review by TakutoIto — CHANGES_REQUESTED** [review]
In omu1p85m.py ,  az0, el0

File: neclib/coordinates/pointing_error/omu1p85m.py (hunk 4 above)
Comment (r-yamada1998) [reply neclib/coordinates/pointing_error/omu1p85m.py:140]: oh, no

Comment (r-yamada1998) [reply neclib/coordinates/pointing_error/omu1p85m.py:140]: My mistake lol

File: neclib/coordinates/pointing_error/pointing_error.py [hunk 5]
```diff
@@ -117,6 +155,35 @@ def offset(self, az: u.Quantity, el: u.Quantity) -> Tuple[u.Quantity, u.Quantity
...
+        self,
+        az: Union[u.Quantity, DimensionLess],
+        el: Union[u.Quantity, DimensionLess],
+        pressure: u.hPa,
+        temperature: u.deg_C,
+        relative_humidity: float,
+        obswl: u.micron,
+    ) -> Tuple[u.Quantity, u.Quantity]:
+        def func(el, A, B):
+            # Zは天体の視位置(天頂角)
```
Comment (takayama-27) [inline neclib/coordinates/pointing_error/pointing_error.py:168]: change to English

**Review by takayama-27 — CHANGES_REQUESTED** [review]
write the comment in  English

**Review by TakutoIto — APPROVED** [review]

**Review by takayama-27 — APPROVED** [review]
LGTM！

**Pull request merged**